.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,26 @@
+name:  Java with Gradle CI
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+
+      - uses: actions/checkout@v3
+
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
+      - name: Run build with Gradle Wrapper
```

Comment:
please separate this block from the previous one with empty line:
```yaml
- name: Setup Gradle
  uses: gradle/gradle-build-action@v2 

- name: Run build with Gradle Wrapper
```

Reply:
✅